doc/Language/unicode.rakudoc

```diff
@@ -102,13 +102,36 @@ Please note that this encoding so far is not supported in the JVM implementation
 
 =head1 Entering unicode codepoints and codepoint sequences
 
-You can enter Unicode codepoints by number (decimal as well as hexadecimal). For example, the character named
-"latin capital letter ae with macron" has decimal codepoint 482 and hexadecimal codepoint 0x1E2:
+=head2 Entering codepoints by number
+
+You can enter Unicode codepoints by number (decimal as well as hexadecimal or octal). For example, the character named
+"latin capital letter ae with macron" has decimal codepoint 482, hexadecimal codepoint 0x1E2, and octal codepoint 0o742:
 
     say "\c[482]"; # OUTPUT: «Ǣ␤»
     say "\x1E2";   # OUTPUT: «Ǣ␤»
+    say "\o742";   # OUTPUT: «Ǣ␤»
+
+Note any character following the hexidecimal and octal examples above will
+be interpreted as part of the conversion unless it is not valid for the number type.
+For example:
+
+    say "\x1E2";   # OUTPUT: «Ǣ␤»
+    say "\x1E2P";  # OUTPUT: «ǢP␤» ('P' is not a hexadecimal digit, result: 2 chars)
+    say "\x1E2 ";  # OUTPUT: «Ǣ ␤» (' ' is not a hexadecimal digit, result: 2 chars)
+    say "\x1E20";  # OUTPUT: «Ḡ␤»  ('0' is a hexidecimal digit', result: 1 char)
+
+The same type of result would occur for the octal version if the ending character
+was not in the set (0..7).
+
+Codepoint sequences can be made in a single string by following a codepoint entry
+with others or intermingled with single characters, words, and spaces as normal
+text. Such codepoint entries are very useful for users who not have ready access to
+a Unicode capable keyboard.
+
+=head2 Entering codepoints by name
 
-You can also access Unicode codepoints by name:
+You can also access Unicode codepoints by name (which may be necessary when a 
+Unicode character has more than one format).
```

Comment:
what does "more than one format" mean?

Reply:
I probably used the wrong words, but I was trying to change the original text which didn't quite fit the revised text. I was thinking of the "Key to the Unicode Code Charts" which mentions  several associated data descriptions. I welcome better wording.

Reply:
sorry i'm not sure what you mean by the "key to the unicode code charts", is that part of the raku docs, or linked to from this file?

Reply:
It's from one of the Unicode Consortium's docs. It's not linked, but I will do so.

Reply:
https://www.unicode.org/charts/About.html i found now it on this page